File: C++/First Steps in Coding - Basics of C++/While Loops/Account Balance.cpp
#include <cstdlib>
#include <iostream>
#include <iomanip>
#include <string>
using namespace std;
int main() {
	double accBalance=0;
	while (true) {
		string command;
		cin >> command;
		if (command == "NoMoreMoney") {
			break;
		}
		else {
			double sum;
			sum = stod(command);
			if (sum < 0) {
				cout << "Invalid operation!" << endl;
				break;
			}
			else {
				accBalance += sum;
				cout << fixed << setprecision(2) << "Increase: " << sum << endl;
			}
		}
	}
	cout << fixed << setprecision(2) << "Total: " << accBalance << endl;
}

File: Python/Basics/While Loop/Lab/AccountBalance.py
balance = 0

while True:
    operation = input()

    if operation != "NoMoreMoney":

        if float(operation) < 0:
            print("Invalid operation!")
            break

        balance += float(operation)
        print(f"Increase: {float(operation):.2f}")

    else:
        break

print(f"Total: {balance:.2f}")